Gemfile

```diff
@@ -39,6 +39,8 @@ gem 'uglifier', '>= 1.0.3'
 gem 'unicorn'
 gem 'validates_formatting_of'
 gem 'will_paginate'
+gem 'elasticsearch-model', '~> 0.1.7'
+gem 'elasticsearch-rails', '~> 0.1.7'
```

Comment:
According to [readme](https://github.com/elastic/elasticsearch-ruby/commit/feca404b5c17994bcd6e4b72e040a93ee47a555c#diff-0ec2e339b7af92418e23e5053790d72bR30), `typhoeus` is recommended for optiomal peformance.

[Here's](https://github.com/elastic/elasticsearch-ruby/blob/9a7f6a9ff26e1010fd274d6feab0ec26e4154f91/elasticsearch-transport/test/integration/transport_test.rb#L19-L26) what seems like an example of using it.